src/site/_data/courses/css/meta.yml

```diff
@@ -1,3 +1,9 @@
+# Used to organize the courses on the /learn page
+date: 2021-05-01
+url: /learn/css/
+draft: true
 title: i18n.courses.learn_css.title
-gradient: "96.46deg, #B323F4 5.23%, #2BA1FF 33.16%, #02E895 68.79%, #6DDB00 90.93%"
-background: "url('/images/courses/learn-css.svg')"
+description: i18n.courses.learn_css.description
+# This path should be relative to the _include folder.
+logo: '../../../images/courses/css/logo.svg'
```

Comment:
Do these work appropriately? I ask cause if they're like the hero images then it should be in the bucket. i doubt there is an issue because the imgix stuff would have thrown an error, but id check to be sure.

Reply:
yeah right now the SVG are local. Let's do a follow up to consider if we want ot CDN them